src/init.cpp

```diff
@@ -628,6 +629,8 @@ void SetupServerArgs()
     gArgs.AddArg("-server", "Accept command line and JSON-RPC commands", ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
     gArgs.AddArg("-rpcallowcors=<host>", "Allow CORS requests from the given host origin. Include scheme and port (eg: -rpcallowcors=http://127.0.0.1:5000)", ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
     gArgs.AddArg("-rpcstats", strprintf("Log RPC stats. (default: %u)", DEFAULT_RPC_STATS), ArgsManager::ALLOW_ANY, OptionsCategory::RPC);
+    gArgs.AddArg("-timestats", strprintf("Log variuous time stats (tx, blocks ..). (default: %u)", DEFAULT_TIME_STATS), ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
```

Comment:
Minor spelling error

```suggestion
    gArgs.AddArg("-timestats", strprintf("Log various time stats (tx, blocks ..). (default: %u)", DEFAULT_TIME_STATS), ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
```